docs/source/examples/pt_zne.md

```diff
@@ -0,0 +1,378 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+```{tags} cirq, zne, pt, basic
+```
+# ZNE with Noise Tailoring
+
+This tutorial helps us understand the cases when noise tailoring is helpful for improving the performance of 
+quantum error mitigation techniques. 
+
+We will
+
+- compare coherent noise to incoherent noise
+- Pauli twirl coherent noise to be incoherent
+- compare the performance of [Zero-Noise Extrapolation](../guide/zne.md) (ZNE)
+on its own and in combination with [Pauli Twirling](../guide/pt.md) (PT) when the circuit is subjected to coherent noise
+
+## Coherent noise vs. Incoherent noise
+
```

Comment:
Add a quick intro before diving in to definitions here. Maybe something like "Noise on quantum devices can broadly be broken into two classes. Each has different properties that requires mitigation by different techniques ..."